api/s2n.h

```diff
@@ -56,6 +56,8 @@ extern int s2n_config_free(struct s2n_config *config);
 extern int s2n_config_free_dhparams(struct s2n_config *config);
 extern int s2n_config_free_cert_chain_and_key(struct s2n_config *config);
 
+extern int s2n_config_init_session_ticket_keys(struct s2n_config *config);
```

Comment:
Why do we need this one? I though adding the key will initialize everything.

Reply:
Oh I meant to add this to s2n_config.h and not in api/s2n.h. Will update.  

Enabling use of session ticket (config->use_tickets) will initialize them (ticket_keys and ticket_key_hashes). You can add a key only if config->use_tickets is enabled. 

s2n_config_init_session_ticket_keys is called in s2n_config_set_session_tickets_onoff.